babel.config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "10"
```

Comment:
it makes sense to bump it to 12 now. Probably, in another PR

Reply:
see https://github.com/appium/appium/issues/15401